contrib/win32/openssh/install-sshd.ps1

```diff
@@ -84,6 +85,28 @@ if (Test-Path $moduliPath -PathType Leaf)
     Repair-ModuliFilePermission -FilePath $moduliPath @psBoundParameters -confirm:$false
 }
 
+#If %programData%/ssh folder already exists, verify permissions and fix, if necessary
+$sshProgDataPath = Join-Path $env:ProgramData "sshTest"
```

Comment:
sshTest -> ssh

Reply:
Please test this once.

Without this change,
1) install the sshd process.
2) enable the file based logging by modifying the sshd_config
3) start the ssh service
4) establish interactive ssh session, sftp session.
5) c:\programdata\ssh folder now has host keys, logs folder (sshd.log, sftp-server.log)
6) run uninstall-sshd.ps1. This doesn't delete the contents of  c:\programdata\ssh folder 

Now run install-sshd.ps1 with your changes. We shouldn't ideally be asked to fix any file permissions.